rhel8/kickstart/ssg-rhel8-ospp-ks.cfg

```diff
@@ -0,0 +1,179 @@
+# SCAP Security Guide OSPP/USGCB profile kickstart for Red Hat Enterprise Linux 7 Server
+# Version: 0.0.2
+# Date: 2015-11-19
```

Comment:
It's probably better to remove the version and date, as it will inherit the upstream version.